CMakeLists.txt

```
@@ -40,6 +40,10 @@ option(MIOPEN_REQ_LIBS_ONLY "Build only the MIOpen required libraries" OFF)
 option(CK_EXPERIMENTAL_BUILDER "Enable experimental builder" OFF)
 option(BUILD_MHA_LIB "Build the static library for flash attention" OFF)
 
+if(CK_EXPERIMENTAL_BUILDER)
+    add_definitions(-DCK_EXPERIMENTAL_BUILDER)
+endif()
+
```

If old CK is compiled with CK_EXPERIMENTAL_BUILDER, it needs access to the experimental headers, which is currently a problem. I've solved this on my local version with the following, but I'm not sure if thats correct for installation purposes etc.

```suggestion
if(CK_EXPERIMENTAL_BUILDER)
    add_definitions(-DCK_EXPERIMENTAL_BUILDER)
    include_directories(${PROJECT_SOURCE_DIR}/experimental/builder/include)
endif()
```

---

Actually this is not good enough, I had to put the include statement higher up in the root cmake file.

---

Thanks. This worked for me and passed CI without this change. I have not figured out whether CK installs headers correctly. I've been looking into that.